base_phone/models/phone_validation_mixin.py

```diff
@@ -5,13 +5,28 @@
 
 from odoo import models
 
+from odoo.addons.phone_validation.tools import phone_validation
+
 
 class PhoneValidationMixin(models.AbstractModel):
-    _inherit = "phone.validation.mixin"
+    _name = "phone.validation.mixin"
```

Comment:
This is not correct. You should inherit the ORM way. You can't replace a whole model.

Reply:
@pedrobaeza the model has been removed in v15. So it can't be inherited.

Reply:
If I had to be nitpicky, I'd probably just want to use different mixin name or not at all. The functions can be called directly instead of having the methods into the models.

Reply:
I see, but you should inherit the new mixin name instead (`mail.thread.phone`), isn't it?:

https://github.com/odoo/odoo/blob/15.0/addons/phone_validation/models/mail_thread_phone.py#L29

Reply:
I don't think so.

This isn't a new mixin, see here: https://github.com/odoo/odoo/blob/14.0/addons/phone_validation/models/mail_thread_phone.py (14.0) 

The main difference between 14.0 and 15.0 is that the mixin for validation has been effectively removed and replaced by methods in this file: 

https://github.com/odoo/odoo/blob/15.0/addons/phone_validation/tools/phone_validation.py

I don't really see any issue other than the mixin should probably get removed completely... On the other hand, it would be hard to make it customizable without the mixin. One would have to monkey patch the phone_validation.py file to extend functionality. That doesn't sound better than restoring the removed mixin.

Reply:
hi @pedrobaeza @llacroix any chances to have it resolved and merged?